h/static/scripts/root-thread.js

```diff
@@ -60,15 +60,29 @@ function RootThread($rootScope, annotationUI, features, searchFilter, viewFilter
 
     var threadFilterFn;
     if (features.flagEnabled('selection_tabs') && !state.filterQuery && state.selectedTab) {
-      threadFilterFn = function (thread) {
-        if (state.selectedTab === uiConstants.TAB_ANNOTATIONS) {
-          return thread.annotation && metadata.isAnnotation(thread.annotation);
-        } else if (state.selectedTab === uiConstants.TAB_NOTES) {
-          return thread.annotation && metadata.isPageNote(thread.annotation);
-        } else {
-          throw new Error('Invalid selected tab');
-        }
-      };
+      if (!features.flagEnabled('orphans_tab')) {
+        threadFilterFn = function (thread) {
+          if (state.selectedTab === uiConstants.TAB_ANNOTATIONS || state.selectedTab === uiConstants.TAB_ORPHANS) {
```

Comment:
If the orphans tab feature flag is not enabled, there should presumably be no way that the orphans tab can be selected? If so, the second condition here is redundant.